src/theme.js

```diff
@@ -8,15 +9,18 @@ const SPACING = {
 };
 
 export default createMuiTheme({
+  palette: {
+    primary: deepPurple,
```

Comment:
How do you feel about pulling one of the shades of purple from https://github.com/mozilla/balrog/blob/master/ui/app/img/balrog.svg for the primary colour? Or will that make the secondary/error colours more difficult to match and/or create other issues?

Reply:
Sure thing. I'll give it a try.